src/de.hpi.swa.graal.squeak/src/de/hpi/swa/graal/squeak/nodes/ExecuteContextNode.java

```diff
@@ -165,8 +168,7 @@ private Object startBytecode(final VirtualFrame frame) {
                         return callPrimitiveNode.primitiveNode.executePrimitive(frame);
                     } catch (final PrimitiveFailed e) {
                         getHandlePrimitiveFailedNode().executeHandle(frame, e.getReasonCode());
-                        LOG.log(Level.FINE, () -> callPrimitiveNode.primitiveNode +
-                                        " (" + ArrayUtils.toJoinedString(", ", FrameAccess.getReceiverAndArguments(frame)) + ")");
+                        LOG.log(Level.FINE, () -> "Failed primitive " + callPrimitiveNode.primitiveNode.getClass().getName());
```

Comment:
I'm assuming you changed this because you saw some multiline string arguments being printed in your console?

Reply:
I also saw Truffle protesting against accessing the frame (throwing exceptions) - I thought it was not worth it